### DISCUSSION TITLE
Test your helix capabilities with a web based shortcut quiz!

### DISCUSSION THREAD

**Discussion by tomgroenwoldt** [helix-editor/helix#6456] [Show and tell]
Hey!

I recently started working on a small web project which tests your knowledge about the editors keyboard shortcuts. I originally just wanted to learn all shortcuts via index cards but realized soon enough that some visuals would benefit the experience. In the near future I want to cover the whole keymap.

[Try it yourself](https://tomgroenwoldt.github.io/helix-shortcut-quiz/) (on a landscape oriented screen)

Source: https://github.com/tomgroenwoldt/helix-shortcut-quiz

Tell me what you think, I am glad for any feedback!

**Comment by tomgroenwoldt** [comment]
Oh, and I completely forgot: I really enjoy this editor! Thanks to everyone who contributes/contributed to this masterpiece :dna:

**Comment by href** [comment]
I like this a lot! Alt shortcuts don't work quite right in my browser (`Alt+.` does not work, but pressing `Alt`, then `.` does). 

In any case, I think this is really useful. I knew most keys by heart already, but it was nice to go through them and I would love to do this again with more categories added, so I can get some practice on the ones I rarely use.

**Reply by tomgroenwoldt** [reply]
I appreciate it :) you are right, some shortcuts trigger browser keybindings and unfortunately I don't think there is anything I can do about that :/

Update: I added a possible `prefix` for a shortcut. This way the `Alt` key in `Alt+.` would already be typed out and shown as a prefix. The downside of this is that you don't get to type the combination as a whole and won't train your muscle memory. This is still better than e.g. closing your current tab via `Ctrl+w` for shortcuts in window mode though.
